grpc-protocol/src/main/java/com/linecorp/armeria/common/grpc/protocol/UnaryGrpcClient.java

```diff
@@ -79,7 +80,8 @@ public UnaryGrpcClient(HttpClient httpClient) {
     public CompletableFuture<byte[]> execute(String uri, byte[] payload) {
         final HttpRequest request = HttpRequest.of(
                 RequestHeaders.of(HttpMethod.POST, uri,
-                                  HttpHeaderNames.CONTENT_TYPE, "application/grpc+proto"),
+                                  HttpHeaderNames.CONTENT_TYPE, "application/grpc+proto",
+                                  HttpHeaderNames.TE, HttpHeaderValues.TRAILERS),
```

Comment:
I didn't add a test case for this because the upstream server does not include `te` header while converting request headers to `MetaData` [here](https://github.com/grpc/grpc-java/blob/v1.22.1/netty/src/main/java/io/grpc/netty/NettyServerHandler.java#L422).